zap/src/main/java/org/parosproxy/paros/network/HttpRequestHeader.java

```diff
@@ -56,6 +56,8 @@
 // ZAP: 2019/06/01 Normalise line endings.
 // ZAP: 2019/06/05 Normalise format/style.
 // ZAP: 2019/12/09 Address deprecation of getHeaders(String) Vector method.
+// ZAP: 2020/11/10 Add convenience method isCss(), refactor isImage() to use new private method
+// isSpecificType(Pattern).
```

Comment:
Looks like you've pulled in other changes :(

Reply:
:( this sign-off thing is driving me nuts. I think I have to reopen the PR from a new clean bunch